network/views.csv

```diff
@@ -0,0 +1,3 @@
+Mount,View,Name,Description,Dip,Azimuth,Start Date,End Date
```

Comment:
A better order might be: Mount,View,Name,Azimuth,Dip,Start Date, End Date, Description

Name is confusing: what does it mean? Is this the target?

Reply:
Yeah agree, thinking has changed between headers. Maybe title? or. Caption?

Reply:
Or... target?